### ISSUE TITLE
InvalidInputProvided: URL did not match a true URL...

### ISSUE THREAD

**Issue by ZeusIssueReporter** [Ekultek/Zeus-Scanner#22]
Error info:
```Traceback (most recent call last):
  File "zeus.py", line 497, in <module>
    err_msg
InvalidInputProvided: URL did not match a true URL...
````

Running details:
`Linux-4.12.0-parrot6-amd64-x86_64-with-Parrot-3.8-JollyRoger`

Commands used:
`zeus.py -b gosuslugi.ru -s -a -x`

Log file info:
```2017-10-01 01:31:09,332;zeus-log;INFO;log file being saved to '/home/seregareal2/Zeus-Scanner/log/zeus-log-4.log'...
2017-10-01 01:31:09,332;zeus-log;INFO;using default search engine...
2017-10-01 01:31:09,332;zeus-log;ERROR;[7;31;31mran into exception 'URL did not match a true URL...' exception has been saved to log file...
Traceback (most recent call last):
  File "zeus.py", line 497, in <module>
    err_msg
InvalidInputProvided: URL did not match a true URL...

2017-10-01 01:31:09,332;zeus-log;INFO;[32mZeus got an unexpected error and will automatically create an issue for this error, please wait...[0m
2017-10-01 01:31:09,333;zeus-log;INFO;[32mgetting authorization...[0m
2017-10-01 01:31:09,354;zeus-log;INFO;[32mextracting traceback from log file...[0m
```

**Comment by Ekultek** [comment]
URL needs either `http://` or `https://` in it. I will create a patch for this later.

**Comment by Ekultek** [comment]
Look for this is the next push

**Comment by Ekultek** [comment]
fixed via 4cf72fc

**Comment by Ekultek** [comment]
```
root@baal-Aspire-5733Z:~/bin/python/zeus-scanner# python zeus.py -b http://gosuslugi.ru -s --verbose
[08:18:22 DEBUG] checking if the application has been run before...
[08:18:22 DEBUG] verifying operating system...
[08:18:23 DEBUG] already ran, skipping...

    __          __________                             __   
   / /          \____    /____  __ __  ______          \ \  
  / /    ______   /     // __ \|  |  \/  ___/  ______   \ \ 
  \ \   /_____/  /     /\  ___/|  |  /\___ \  /_____/   / / 
   \_\          /_______ \___  >____//____  >          /_/  
                       \/   \/           \/  v1.0.25.b683(revision)
        https://github.com/ekultek/zeus-scanner.git
                Advanced Dork Searching...


[*] starting up at 08:18:23..


[08:18:23 DEBUG] running with options '{'runInVerbose': True, 'runSqliScan': True, 'spiderWebSite': 'http://gosuslugi.ru'}'...
[08:18:23 INFO] log file being saved to '/home/baal/bin/python/zeus-scanner/log/zeus-log-115.log'...
[08:18:23 DEBUG] using default search engine (Google)...
[08:18:23 INFO] using default search engine...
[08:18:23 INFO] starting blackwidow on 'http://gosuslugi.ru'...
[08:18:23 DEBUG] testing connection to the URL...
[08:18:25 DEBUG] connection satisfied, continuing process...
[08:18:26 INFO] found a total of 29 unique usable URL's, saved to '/home/baal/bin/python/zeus-scanner/log/blackwidow-log/blackwidow-log-1.log'...
[08:18:26 PROMPT] would you like to process found URL: 'https://gu-st.ru/htdocs/js/lib/oclazyload.js?fh=2022b62e59d2af0bce6612944365ae27'[y/N]: y
[08:18:56 INFO] creating arguments for sqlmap...
[08:18:56 PROMPT] start the sqlmap API server and press enter when ready... 
[08:18:58 INFO] initializing new sqlmap scan with given URL 'https://gu-st.ru/htdocs/js/lib/oclazyload.js?fh=2022b62e59d2af0bce6612944365ae27'...
[08:18:58 DEBUG] scan initialized...
[08:18:58 INFO] gathering sqlmap API scan ID...
[08:18:58 DEBUG] current sqlmap scan ID: '777f442a0ef2c403'...
[08:18:58 INFO] starting sqlmap scan on url: 'https://gu-st.ru/htdocs/js/lib/oclazyload.js?fh=2022b62e59d2af0bce6612944365ae27'...
[08:18:58 DEBUG] using arguments: '{}'...
[08:18:58 INFO] adding arguments to sqlmap API...
[08:18:58 WARNING] please keep in mind that this is the API, output will not be saved to log file, it may take a little longer to finish processing, and you will need to restart the sqlmap API server after each iteration, launching sqlmap...
------------------------------
sqlmap> [08:19:00 INFO] testing connection to the target URL
sqlmap> [08:19:01 INFO] checking if the target is protected by some kind of WAF/IPS/IDS
sqlmap> [08:19:02 INFO] testing if the target URL is stable
sqlmap> [08:19:03 INFO] target URL is stable
sqlmap> [08:19:03 INFO] testing if GET parameter 'fh' is dynamic
sqlmap> [08:19:04 WARNING] GET parameter 'fh' does not appear to be dynamic
sqlmap> [08:19:05 WARNING] heuristic (basic) test shows that GET parameter 'fh' might not be injectable
sqlmap> [08:19:05 INFO] testing for SQL injection on GET parameter 'fh'
sqlmap> [08:19:06 INFO] testing 'AND boolean-based blind - WHERE or HAVING clause'
^C[08:19:08 ERROR] user aborted process...


[*] shutting down at 08:19:08..


root@baal-Aspire-5733Z:~/bin/python/zeus-scanner# 
```
